frontend/src/clients/apiClient.ts

```diff
@@ -21,24 +21,75 @@ export interface Species {
 
 export interface Sighting {
   id: number;
-  seenBy: string;
-  seenOn: string;
   species?: Species;
-  imageUrl: string;
-  description: string;
+  seenBy: string;
+  seenOn: Date;
+  imageUrl?: string;
+  description?: string;
   whaleCount: number;
-  confirmationStatus: string;
-  location: string;
+  location?: string;
   latitude: number;
   longitude: number;
 }
 
+export interface PendingRequest {
+  sightingId: number;
+  confirmationStatus: number;
+}
+
+export interface ConfirmOrRejectRequest {
+  NewConfirmationStatus: number;
+}
+
+export interface ErrorResponse {
+  sightingId: number;
+  errorMessage: any;
+}
+
 export const getAllSpecies = async (): Promise<Species[]> => {
   const response = await fetch(`${backendUrl}/whales`);
   const whaleListResponse: ListResponse<Species> = await response.json();
   return whaleListResponse.items;
 };
 
+export const getAllPendingSightings = async (): Promise<Sighting[]> => {
+  const response = await fetch(`${backendUrl}/sightings/pending`);
+  const pengingSighting: ListResponse<Sighting> = await response.json();
+  return pengingSighting.items;
+};
+
+export const confirmOrRejectSighting = async (
+  sightingId: number,
+  confirmationRequest: ConfirmOrRejectRequest,
```

Comment:
Since there's only one field, you can simplify this to
```suggestion
  newStatus: number,
```

However, I think I'd much prefer:

1. Add a type like
```
type ConfirmationStatus: "pending" | "approved" | "rejected";
```
2. Add a mapping object from string to number:
```
const statusMapping: { [key: ConfirmationStatus]: number } = {
  pending: 0,
  approved: 1,
  rejected: 2,
}
```
This variable can be defined inside the `confirmOrRejectSighting` function, since it's only used there (my syntax might be wrong and I might have got the actual number mapping wrong, so double check)

3. Take a string as a parameter:
```suggestion
  newStatus: ConfirmationStatus,
```
4. Map to a number just before you send off the fetch request!
```
const newStatusNumber = statusMapping[newStatus];
```

Make sure to change how the function is called elsewhere (you'll get red underlines after changing the parameter type so it shouldn't be too hard!)